tests/mockLogzioListener/persistentFlags.py

```diff
@@ -0,0 +1,18 @@
+# Modules imported only once, this is effectively a singleton
+
+
+class PersistentFlags:
```

Comment:
saw this,a nice python palavra
@property
https://www.programiz.com/python-programming/property